client/cypress.json

```
@@ -0,0 +1,12 @@
+{
+  "baseUrl": "http://localhost:4200/",
+  "fileServerFolder": ".",
+  "fixturesFolder": "cypress/fixtures",
+  "integrationFolder": "cypress/integration",
+  "pluginsFile": "cypress/plugins/index.ts",
+  "supportFile": false,
```

See this earlier thread: https://github.com/UMM-CSci-3601/3601-iteration-template/pull/119#discussion_r579720794

```suggestion
  "supportFile": "cypress/support/index.ts",
```